style.css

```diff
@@ -0,0 +1,80 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+.container {
+  display: flex;
+  justify-content: space-around;
+  width: 375px;
+  height: 48 px;
+}
+
+.bigger-text {
+  display: flex;
+  align-items: center;
+  width: 327px;
+  height: 150px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 40px;
+  color: #172b4d;
+  padding: 80px 24px 0 24px;
+}
+
+.smaller-text {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  width: 327px;
+  height: 146px;
+  color: 344563;
+  padding: 8px 24px 0 24px;
+}
+
+.second-container {
+  background-image: url(mobile-background.png);
+  height: 100vh;
+}
+
```

Comment:
- Kindly make sure that your background image is well  disposed and is not repeated to do that i can suggest you to add that property I listed below to this class :+1: 



```
.second-container {
  background-image: url(mobile-background.png);
  background-repeat: no-repeat;
  background-size: cover;
  min-height: calc(100vh - 599px);
  border-bottom-left-radius: 90px;
  background-color: #fff;
}
```